dokka-subprojects/plugin-base/src/main/resources/dokka/styles/style.css

```diff
@@ -398,10 +378,6 @@ code:not(.block) {
     vertical-align: bottom;
 }
 
-.symbol > a {
-    color: var(--hover-link-color);
-}
```

Comment:
This style seems to be outdated. 
Now:
<img width="845" alt="Screenshot 2025-02-06 at 11 03 43" src="https://github.com/user-attachments/assets/4a0b8fc7-02ea-4ecf-b576-d295fc0a8ba1" />
If the style applied:
<img width="844" alt="Screenshot 2025-02-06 at 11 06 35" src="https://github.com/user-attachments/assets/1b880ddf-c798-4830-99ba-8790091a24bd" />